readme.md

```diff
@@ -26,7 +26,7 @@ git clone --recursive https://github.com/Azure/azure-iot-sdk-csharp.git
 
 | Features                                                                                                         | mqtt                | mqtt-ws             | amqp                | amqp-ws             | https               | Description                                                                                                                                                                                                                                                            |
 |------------------------------------------------------------------------------------------------------------------|---------------------|---------------------|---------------------|---------------------|---------------------|------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------|
-| [Authentication](https://docs.microsoft.com/en-us/azure/iot-hub/iot-hub-security-deployment)                     | :heavy_check_mark:* | :heavy_check_mark:* | :heavy_check_mark:* | :heavy_check_mark:* | :heavy_check_mark:* | Connect your device to IoT Hub securely with supported authentication, including private key, SASToken and X-509 Self Signed.  *This SDK does not support X-509 CA Signed at the moment.                                                                               |
+| [Authentication](https://docs.microsoft.com/en-us/azure/iot-hub/iot-hub-security-deployment)                     | :heavy_check_mark: | :heavy_check_mark:* | :heavy_check_mark: | :heavy_check_mark:* | :heavy_check_mark:* | Connect your device to IoT Hub securely with supported authentication, including private key, SASToken, X-509 Self Signed and X-509 CA Signed. *IoT Hub only supports X-509 CA Signed over AMQP and MQTT at the moment.                                                                               |
```

Comment:
I think what this should say is that the _IoT Hub service_ doesn't support uploading a CA signer certificate then allowing devices holding a certificate issued by this CA to authenticate.

CA-signed certs or self-signed work fine for all protocols as long as each device certificate is uploaded to the IoT Hub service.

/cc @tameraw @mastermanu

Reply:
Seems like we should point to a document to explain the details @eustacea